src/components/Field/Field.scss

```diff
@@ -57,7 +61,7 @@
   }
 
   .input {
-    margin: 13px 20px 0 0;
+    // margin: 13px 20px 0 0;
```

Comment:
This can be removed I guess

Reply:
Done https://github.com/parse-community/parse-dashboard/pull/1708/commits/16010c7db70c04f249edafd95bf313fe8a195461 !